node/src/main/scala/coop/rchain/node/BlockStoreMigrate.scala

```diff
@@ -0,0 +1,58 @@
+package coop.rchain.node
+
+import coop.rchain.blockstorage.{FileLMDBIndexBlockStore, KeyValueBlockStore}
+import coop.rchain.casper.storage.RNodeKeyValueStoreManager
+import coop.rchain.metrics.Metrics
+import coop.rchain.rspace.Context
+import coop.rchain.shared.Log
+import coop.rchain.shared.syntax._
+import monix.eval.Task
+import org.rogach.scallop.ScallopConf
+import monix.execution.Scheduler.Implicits.global
+
+import java.nio.file.Path
+
+final case class Options(arguments: Seq[String]) extends ScallopConf(arguments) {
+  val width = 120
+  helpWidth(width)
+  printedName = "migrate-blockstore"
+
+  val originalBlockStore = opt[Path](
+    descr = "original block store directory",
+    required = true
+  )
+
+  val targetDataDir = opt[Path](
+    descr = "Output block store directory",
+    required = true
+  )
+
+  verify()
+
+}
+object BlockStoreMigrate {
+  def main(args: Array[String]): Unit = {
```

Comment:
How we run multiple _main_ methods?

Reply:
A separate binary would be generated. We can run through docker.

Reply:
How this additional binary is generated? Maybe it's easier to just extend existing commands as I suggested in the previous comment https://github.com/rchain/rchain/pull/3342#pullrequestreview-602022629?

Reply:
it is with `rnode`. Nothing special just another binary like `rnode`